templates/_layouts/docs.html

```diff
@@ -47,7 +47,7 @@
 
             <ul class="p-side-navigation__list">
               <li class="p-side-navigation__item--title"><span class="p-side-navigation__text">Base elements</span></li>
-              {{ side_nav_item("/docs/base/code", "Code") }}
+              {{ side_nav_item("/docs/base/code", "Code", "updated") }}
```

Comment:
While at it, could you remove other labels from side nav, as they will not be relevant in next release?